<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PySystemTrader V0.1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>721</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>키움 증권 System Trader V 0.1 입니다.</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_Order">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>52</y>
      <width>231</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>수동주문</string>
    </property>
    <widget class="QLabel" name="label_Account">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>24</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>계좌</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_OrderType">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>54</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>주문</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Code">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>85</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>종목</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_OrderType2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>135</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>종류</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Ammount">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>159</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>수량</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Price">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>192</y>
       <width>56</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>가격</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Account">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>18</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_Amount">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>157</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_Price">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>187</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>2000000</number>
     </property>
     <property name="singleStep">
      <number>50</number>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_Hoga">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>130</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>지정가</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>시장가</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Code">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_CodeName">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>103</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:RGB(170, 255, 255)</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Order">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>220</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>현금주문</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_OrderType">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>신규매수</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>신규매도</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>매수취소</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>매도취소</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableWidget_AccountInfo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>771</width>
      <height>51</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>예수금(D+2)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>총매입</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>총평가</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>총손익</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>총수익율(%)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>추정자산</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tableWidget_DetailCodeInfo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>410</y>
      <width>771</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>종목명</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>보유량</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>매입가</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>현재가</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>세금 및 수수료</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>평가손익</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>수익율 (%)</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="pushButton_GetBalanceInfo">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>323</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>조회</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_RealTime">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>실시간 조회</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_Terminal">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>531</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(175, 175, 175)</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Check">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>조회</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_RealTime_codeinfo">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>24</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>실시간 조회</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_Terminal_RealTime">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>200</y>
      <width>531</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(175, 175, 175)</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
